crates/feldera-types/src/transport/s3.rs

```diff
@@ -5,6 +5,10 @@ fn default_max_concurrent_fetches() -> u32 {
     8
 }
 
+fn default_max_retries() -> u32 {
+    3
```

Comment:
I vote for a higher default. I've seen S3 timeout two or three times in a row. So 5 or even 10 might be a safer value. (sorry I may have introduced the confusion in the first place).